spec/spec_helper.rb

```diff
@@ -10,7 +10,6 @@
 require 'voxpupuli/test/spec_helper'
 
 RSpec.configure do |c|
-  c.facterdb_string_keys = false
```

Comment:
This Optik is coming from modulesync_config : https://github.com/voxpupuli/puppet-autofs/blob/master/.sync.yml#L13 https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb.erb#L13

Reply:
Or just that this would get wiped out by a future modulesync and should just close this PR and leave it as it is?

Reply:
https://github.com/voxpupuli/modulesync_config/blob/78c09c9954deb7b2510a381a1dc750a9feba1855/config_defaults.yml#L115

Ah, default from here. Yeah, will just leave this as is